smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/CommandGenerator.java

```diff
@@ -299,6 +299,12 @@ private void generateEndpointParameterInstructionProvider() {
                 }
                 paramNames.add(name);
             });
+
+            parameterFinder.getOperationContextParamValues(operationInput).forEach((name, jmesPathForInputInJs) -> {
+                writer.write(
+                   "$L: { type: \"operationContextParams\", name: $L },",
```

Comment:
```suggestion
                   """
                   $L: { type: "operationContextParams", name: $L },
                   """,
```

optional to avoid escaping double quotes

Reply:
Added in [`0d5f950` (#1379)](https://github.com/smithy-lang/smithy-typescript/pull/1379/commits/0d5f950ce46c8ef019aacbd4381793575116d3fc)